packages/ai-core/src/browser/ai-configuration/agent-configuration-widget.tsx

```diff
@@ -137,14 +137,26 @@ export class AIAgentConfigurationWidget extends ReactWidget {
                     Enable Agent
                 </label>
             </div>
+            <div className="settings-section-subcategory-title ai-settings-section-subcategory-title">
+                Prompt Templates
+            </div>
             <div className='ai-templates'>
-                {agent.promptTemplates?.map(template =>
-                    <TemplateRenderer
-                        key={agent?.id + '.' + template.id}
-                        agentId={agent.id}
-                        template={template}
-                        promptCustomizationService={this.promptCustomizationService} />)}
+                {agent.promptTemplates
+                    ?.filter(template => !template.variantOf)
```

Comment:
what if the agent does not provide a template without a variant? shouldn't we handle this? eg show 'no default template available'?

Reply:
![image](https://github.com/user-attachments/assets/5988df54-22e4-4f44-9e5d-51bfca1fc4c2)
Added

Reply:
Great suggestions, see above